### DISCUSSION TITLE
one image generating - switch to small

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3176] [Q&A] [answered]
ok, i try to explain ....

if i generate one image its more or less full screen in that window for pictures
if its ready it switch to a smaller size ... so i must "klick" it again to see it full and than i can save it ...
every time...
why you dont leave it in full so i can save it instantly  ^^

**Comment by mashb1t** [comment, marked as answer]
This is due to consistency between one and many images. You can find the same request in https://github.com/lllyasviel/Fooocus/pull/2936.

**Reply by kalle07** [reply]
too bad ^^
